src/main/webapp/js/components/csv/csv-config-page.jsx

```diff
@@ -67,6 +68,12 @@ class CsvConfigPage extends Component {
         required: fieldConfig ? fieldConfig.required : false,
         format: fieldConfig ? fieldConfig.format : null,
       },
+      {
+        name: 'keyField',
+        fieldType: BOOLEAN,
+        displayName: 'Key Field?',
+        required: true,
```

Comment:
boolean fields should not be required, it will always be true or false (or at least it should be)